openapi3/loader_exmaple_schema_ref_test.go

```diff
@@ -0,0 +1,17 @@
+package openapi3
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestLoadSpecWithExampleRefToSchemas(t *testing.T) {
+	loader := NewLoader()
+	loader.IsExternalRefsAllowed = true
+	api, err := loader.LoadFromFile("testdata/example-schemas-ref.json")
+	exampleValue := api.Paths["/pet"].Put.Responses.Get(400).Value.Content.Get("*/*").Examples["200"].Value.Value
+	schemaRef := api.Components.Schemas["Pet"].Value
+	require.Equal(t, exampleValue, schemaRef)
+	require.NoError(t, err)
```

Comment:
Check err where it is set.

Also, use `doc` instead of `api` as is done in the other tests.

Also, ensure document is valid per `doc.Validate(ctx)` as is done in the other tests.

Reply:
fixed